File: app/src/main/java/parkhomov/andrew/lensthicknesscalculator/activities/main/MyApp.java
package parkhomov.andrew.lensthicknesscalculator.activities.main;

import android.annotation.SuppressLint;
import android.app.Application;
import android.content.Context;

import com.crashlytics.android.Crashlytics;

import io.fabric.sdk.android.BuildConfig;
import io.fabric.sdk.android.Fabric;
import parkhomov.andrew.lensthicknesscalculator.R;
import parkhomov.andrew.lensthicknesscalculator.activities.utils.CONSTANT;
import uk.co.chrisjenx.calligraphy.CalligraphyConfig;

/**
 * Created by MyPC on 28.07.2017.
 */

public class MyApp extends Application {

    @SuppressLint("StaticFieldLeak")
    public static Context context;

    public static Context getAppContext() {
        return context;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        context = getApplicationContext();
        if (!BuildConfig.DEBUG)
            Fabric.with(this, new Crashlytics());

        CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
                .setDefaultFontPath("raleway.ttf")
                .setFontAttrId(R.attr.fontPath)
                .build()
        );
    }
}

File: app/src/main/java/parkhomov/andrew/lensthicknesscalculator/activities/main/MyApp.kt
package parkhomov.andrew.lensthicknesscalculator.activities.main

import android.app.Application
import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.BuildConfig
import io.fabric.sdk.android.Fabric
import parkhomov.andrew.lensthicknesscalculator.R
import uk.co.chrisjenx.calligraphy.CalligraphyConfig

/**
 * Created by MyPC on 28.07.2017.
 */

class MyApp : Application() {

    override fun onCreate() {
        super.onCreate()
        getAppContext = this
        if (!BuildConfig.DEBUG)
            Fabric.with(this, Crashlytics())

        CalligraphyConfig.initDefault(CalligraphyConfig.Builder()
                .setDefaultFontPath("raleway.ttf")
                .setFontAttrId(R.attr.fontPath)
                .build()
        )
    }


    companion object {
        lateinit var getAppContext: MyApp
            private set
    }
}
